back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -279,4 +279,33 @@
       end
     end
   end
+
+  context 'not logged in' do
+    before do
+      @projects = ['published','archived','draft','published','archived']
+        .map { |ps|  create(:project, admin_publication_attributes: {publication_status: ps})}
```

Comment:
Could you configure Rubocop? No big deal definitely, just a tiny note :)

Reply:
I have Rubocop. Thanks for reminding me to use it :+1: